modules/chapter2/pages/rbac.adoc

```diff
@@ -0,0 +1,138 @@
++
+[subs=+quotes]
+----
+catalog:
+      providers:
+        githubOrg:
+          default:
+            id: development
+            orgUrl: ${GITHUB_ORG_URL}
+*permission:
+      enabled: true
+      rbac:
+        admin:
+          users:
+            - name: user:/default/rsriniva*
+----
+
+. Re-deploy the `rhdh` helm chart for the changes to take effect.
+
+. Sign out from the existing RHDH session and login again using the GitHub ID of the declared policy administrator.
+
+. Navigate to the `Catalog` page in RHDH, and click `CREATE`. Note that you are not allowed to create new components.
+
+. Next, try navigating to the `API` page and click on `REGISTER EXISTING API`. You will be shown an `ERROR 404` page. With RBAC enabled, most features are disabled by default. You need to explicitly enable permissions to resources in RHDH.
++
+image::404-error-rbac.png[title=Not allowed to register APIs]
+
+## Lab: Declarative Policy Definition in CSV Files
+
+To selectively allow RHDH features, you need to declare `roles` with allowed policies, and then assign these roles to users or groups.
+
+. Edit the `app-config-rhdh` ConfigMap, and add a reference to the policy CSV file under the `rbac` attribute that you enabled previously:
++
+[subs=+quotes]
+----
+catalog:
+      providers:
+        githubOrg:
+          default:
+            id: development
+            orgUrl: ${GITHUB_ORG_URL}
+permission:
+      enabled: true
+      rbac:
+        *policies-csv-file: ./rbac/rbac-policy.csv*
+        admin:
+          users:
+            - name: user:/default/rsriniva
+----
+
+. You will create a new ConfigMap to store the contents of the CSV policy file. Create a new ConfigMap named `rhdh-rbac-policy` with the following content. These policies allow the user with the `admins` role to read, create and update components, but fetching catalog information from remote locations is denied:
++
+[subs=+quotes]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: rhdh-rbac-policy
+  namespace: devhub
+immutable: false
+data:
+  rbac-policy.csv: |-
+    p, role:default/admins, catalog-entity, read, allow
+    p, role:default/admins, catalog.entity.create, create, allow
+    p, role:default/admins, catalog.entity.refresh, update, allow
+    p, role:default/admins, catalog.location.create, create, deny
+    p, role:default/admins, catalog.location.read, read, deny
+    g, user:default/rsriniva, role:default/admins
+----
+
+. To mount the contents of the RBAC policy CSV file into the RHDH container, edit the `rhdh` helm chart in `Form View` and follow the instructions as outlined in https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.0/html-single/administration_guide_for_red_hat_developer_hub/index#mounting-literal-policy-csv-literal-file-to-the-developer-hub-helm-chart. Ensure that the paths and name of ConfigMap match your set up.
```

Comment:
I'd explicitly add the path into this setting in the from view like done for the other example settings so far. so root schema > backstage chart schema > ...

Reply:
also maybe adding that you add an additional mount path without touching the already existing one, just to make sure users won't change that one :)

Reply:
the docs are pretty clear about form navigation in helm chart. This one is too cluttered and verbose to rewrite in our instructions.

Reply:
added a warning about not deleting existing volume mounts